app/serializers/schema_blogpost_serializer.rb

```diff
@@ -0,0 +1,63 @@
+# frozoen_string_literal: true
+
+class SchemaBlogpostSerializer < SchemaBaseSerializer
+  attr_reader :entry
+
+  def initialize(context, entry)
+    super(context)
+    @entry = entry
+  end
+
+  def to_jsonld_data
+    base.merge(main_entry, author, publisher).to_json
+  end
+
+  def base
+    {
+        '@context': 'http://schema.org',
+        '@type': 'BlogPosting',
+        'headline': entry.title,
+        'alternativeHeadline': 'MagmaLabs Today I Learned',
+        'datePublished': entry.created_at.strftime('%Y-%m-%d'),
+        'dateModified': entry.updated_at.strftime('%Y-%m-%d'),
+        'articleBody': entry.body,
+        'image': context.image_url('logo'),
+        'inLanguage': 'en-US',
+        'copyrightYear': entry.created_at.year,
+        'keywords': %w{#{entry.channel_name} MagmaLabs TIL BlogPost},
```

Comment:
> Keywords or tags used to describe this content. Multiple entries in a keywords list are typically delimited by commas.

Hey, @softr8 . Do you know if does it affect not using commas here? 🤔 
Not sure tbh, but would be nice to know

Reply:
I found this to be ambiguous everywhere, but most of them were using an array